tools/code_format/config.yaml

```diff
@@ -172,6 +172,7 @@ paths:
     - source/extensions/compression/zstd/common/dictionary_manager.h
     - source/extensions/filters/http/adaptive_concurrency/controller
     - source/extensions/filters/http/admission_control
+    - source/extensions/filters/http/api_key_auth
```

Comment:
This is necessary because the `ExceptionFreeFactoryBase` cannot handle the case where the route specific configuration loading will throw the exception.